lib/menu.ts

```diff
@@ -190,18 +214,22 @@ function findAncestorChild(ancestor: Element, elem: Element|null): Element|null
 /**
  * Implements a menu item which opens a submenu.
  */
-export function menuItemSubmenu(submenu: MenuCreateFunc, ...args: DomElementArg[]): Element {
+export function menuItemSubmenu(
+  submenu: MenuCreateFunc,
+  options: ISubMenuOptions,
+  ...args: DomElementArg[]
+): Element {
   const ctl: PopupControl<IMenuOptions> = PopupControl.create(null);
 
-  const popupOptions: IMenuOptions = {
+  const popupOptions: IMenuOptions = Object.assign(options, {
```

Comment:
Using `{ placement etc, isSubMenu: true, ...options }` would make more sense to me: the options passed-in should overwrite the hard-coded defaults.